lib/WorkflowMain.groovy

```diff
@@ -74,7 +74,7 @@ class WorkflowMain {
         NfcoreTemplate.awsBatch(workflow, params)
 
         // Check input has been provided
-        if (!params.input) {
+        if (!params.input && !params.pep) {
```

Comment:
Again, should be be solely `--input`, which we then accept files with `*.{csv,yml,yaml}`